packages/cli/test/bin/bin.test.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-unused-vars */
+// packages/cli/test/bin.parseargs.test.js
+import { spawn } from "node:child_process";
+import { strict as assert } from "node:assert";
+import { fileURLToPath } from "node:url";
+import { dirname, join } from "node:path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const binPath = join(__dirname, "../../src/bin.js");
+
+function runCLI(args = [], timeoutMs = 10000) {
+  return new Promise((resolve) => {
+    const proc = spawn(process.execPath, [binPath, ...args], { env: process.env });
+
+    let stdout = "";
+    let stderr = "";
+    let timedOut = false;
+
+    const to = setTimeout(() => {
+      timedOut = true;
+
+      try {
+        proc.kill();
+      } catch (e) {
+        /* empty */
+      }
+    }, timeoutMs);
+
+    proc.stdout.on("data", (d) => {
+      stdout += d.toString();
+    });
+    proc.stderr.on("data", (d) => {
+      stderr += d.toString();
+    });
+
+    proc.on("close", (code) => {
+      clearTimeout(to);
+      resolve({ stdout, stderr, code, timedOut });
+    });
+
+    // in case spawn errors synchronously (rare)
+    proc.on("error", (err) => {
+      clearTimeout(to);
+      resolve({ stdout, stderr: String(err), code: 1, timedOut });
+    });
+  });
+}
+
+(async () => {
+  console.log("Running parseArgs CLI tests...");
+
+  // 1) --help should include Usage and Commands
+  {
+    const r = await runCLI(["--help"]);
+    assert.ok(!r.timedOut, "--help timed out");
+    assert.ok(/Usage:/.test(r.stdout) || /usage:/.test(r.stdout), "help should include Usage");
```

Comment:
This is kind of a missed opportunity to use regex for what it's good at. Why not `/[Uu]sage:/` instead? But also, if we know the exact help string we're expecting, why be loose on the capitalization to begin with?